### ISSUE TITLE
Getting Pig Food delivered

### ISSUE THREAD

**Issue by Comatica** [Courseplay/courseplay#4624]
Using Courseplay to deliver Pig Food directly to the Pig Sty and getting an error that Pig Food is not accepted.  Helper drives right past tip point and shows an error in the log saying filltype 48 not supported by tip point. yet if i tip manually i can feed pig food to the pigs... Not sure where I can add filltype 48 in the files as I do not know coding.  below if the pastebin to the log where I found the information.  The log shows relevant information around line 1150 forward (geez that log had alot of information in it...)

https://pastebin.com/cP5XcrEc

**Comment by Tensuko** [comment]
Just a sidenote.... read the Mods Name before put it into mos Folder:
Error: Failed to open xml file 'C:/Users/comat/Documents/My Games/FarmingSimulator2019/mods/jdseeders_pack_unzip/modDesc.xml'.
Load mod: jdseeders_pack_unzip (Hash: 6d6a8e303856f15c58e3de1e0ec92888)
Error: Missing descVersion attribute in mod jdseeders_pack_unzip

Now to your Problem:
You got a Callstack from husbandryPiglarge, wich is a Pig farm I would say and maybe the one you use, also in combination with FS19_AnimalPenExtension, maybe the pigfarm did not implement the penextension in the right way.

2019-12-04 04:20 C:/Users/comat/Documents/My Games/FarmingSimulator2019/mods/fs19_huge_pig_enclosure/husbandryPigLarge.i3d (34.64 ms)
2019-12-04 04:20   Error: Failed to find child 0 from node pigTrigger, only 0 childs given
2019-12-04 04:20 LUA call stack:
2019-12-04 04:20   dataS/scripts/i3d/I3DUtil.lua (20) : printCallstack
2019-12-04 04:20   dataS/scripts/i3d/I3DUtil.lua (88) : checkChildIndex
2019-12-04 04:20   dataS/scripts/placeables/Placeable.lua (518) : indexToObject
2019-12-04 04:20   dataS/scripts/animals/AnimalHusbandry.lua (75) : load
2019-12-04 04:20   =C:/Users/comat/Documents/My Games/FarmingSimulator2019/mods/FS19_AnimalPenExtension/scripts/misc/APE_ChangedFunctions.lua (82) : superFunc
2019-12-04 04:20   dataS/scripts/placeables/Placeable.lua (967) : load
2019-12-04 04:20   dataS/scripts/animals/AnimalHusbandry.lua (228) : loadFromXMLFile
2019-12-04 04:20   =C:/Users/comat/Documents/My Games/FarmingSimulator2019/mods/FS19_AnimalPenExtension/scripts/misc/APE_ChangedFunctions.lua (176) : superFunc
2019-12-04 04:20   dataS/scripts/missions/mission00.lua (671) : loadFromXMLFile
2019-12-04 04:20   dataS/scripts/misc/AsyncManager.lua (42) : lambda
2019-12-04 04:20   dataS/scripts/misc/AsyncManager.lua (119) : runLambda
2019-12-04 04:20   dataS/scripts/misc/DeferredLoadingManager.lua (39) : runTopTask
2019-12-04 04:20   dataS/scripts/main.lua (1856) : update
2019-12-04 04:20   Error: Index not found: 0|17|0
2019-12-04 04:20   Error: Failed to find child 0 from node pigTrigger, only 0 childs given

CP checks for the exactly filltype, but it could be that you manually can fill up with pigfood, but the farm accept it and display it as something else. (Like in display is shown Maize/Soyabean. But then you would have to add the Type 48 to the Trigger of the PigFarm Mod, I think in the editor.

**Comment by Comatica** [comment]
Thank you Tensuko for taking the time to assist me today, I appreciate it.

I have been looking in the xml file for the pig farm both stock and the modded one I was going to originally use (fs19_huge_pig_enclosure.zip) to find out where the triggers filltypes are specified so I could add 48, however I do not seem to see those values in the .xml file.  I suspect it is a bit more complicated than editing a text file.  I tried opening the .i3d but those appear to be just editing the appearance in game via the giants editor.

As I stated I am very much a novice when it comes to editing mods and game files, but if it is possible to do I would be forever in debt for the guidance to modify the correct file in the correct way to get this to work.

Again thank you for your time.
